web/Gruntfile.js

```diff
@@ -9,6 +9,10 @@
 
 module.exports = function (grunt) {
 
+  // Add new languages here
+  // Do not put en or exclaim in this list
+  var i18nForeignLanguages = ['app/i18n/ar.json'];
```

Comment:
Pretty sure you're correct. I'll go ahead and make that change. It'll be easy to rename it back later if we want it to be more general.